src/overlays/actors/ovl_Bg_Jya_Bombchuiwa/z_bg_jya_bombchuiwa.h

```diff
@@ -6,9 +7,16 @@
 
 struct BgJyaBombchuiwa;
 
+typedef void (*BgJyaBombchuiwaActionFunc)(struct BgJyaBombchuiwa*, GlobalContext*);
+
 typedef struct BgJyaBombchuiwa {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x6C];
+    /* 0x014C */ BgJyaBombchuiwaActionFunc actionFunc;
+    /* 0x0150 */ ColliderJntSph collider;
+    /* 0x0170 */ ColliderJntSphItem colliderItems;
+    /* 0x01B0 */ f32 lightRayIntensity;
+    /* 0x01B4 */ s16 timer;
+    /* 0x01B6 */ u8 drawFlag; // Upper 3 bits are always 0. lower 4 bytes used to determine how the actor is drawn.
```

Comment:
Since this field stores multiple flags, I suggest using a plural name. Also I don't think it's necessary to mention some bits are never set since that's kind of implicit when using a certain amount of flags.
```suggestion
    /* 0x01B6 */ u8 drawFlags; // used to determine how the actor is drawn
```